index.bs

```diff
@@ -591,8 +662,35 @@ with parameters |session| and |capabilities| is:
 
 </div>
 
+Common Data Types {#data_types}
+===============================
+
+This section defines data types which are common to many modules.
+
+## Browsing Context ## {#data_types-browsing_context}
+
+[= remote end schema=] and [= local end schema=]
+```
+BrowsingContext = text;
+```
+
+Each [=/browsing context=] has an associated <dfn export>browsing context id</dfn>,
+which is a string uniquely identifying that browsing context. For browsing
+contexts with an associated WebDriver [=window handle=] the [=/browsing context
+id=] must be the same as the [=window handle=].
+
+## Realm ## {#data_types-realm}
+
+[= remote end schema=] and [= local end schema=]
+```
+Realm = text;
```

Comment:
`Realm` appears to be unused elsewhere. I believe that's known and not an accident, just pointing it out to be sure.